app/main.py

```diff
@@ -1 +1,79 @@
-# write your code here
+import os
+
+
+BUFFER_SIZE = 2048
+
+
+def move_file(user_command: str) -> None:
+    character_separator = "/"
+
+    class FileInfo:
+        def __init__(self, file_name: str) -> None:
+            self.path = []
+            self.full_path = ""
+            self.path_only = False
+            self.filename = ""
+            self.full_filename = ""
+
+            if file_name[-1] == character_separator:
+                self.path = file_name[:-1].split(character_separator)
+                self.full_path = file_name.rstrip(character_separator)
+                self.path_only = True
+            else:
+                self.full_filename = file_name
+                temp = file_name.split(character_separator)
+                self.filename = temp[-1]
+                self.path = temp[:-1]
+                self.full_path = character_separator.join(self.path)
+
+        def path_add_filename(self, filename: str) -> str:
+            if self.path and self.path_only:
+                return character_separator.join((self.full_path, filename))
+            return self.full_filename
+
+    command = user_command.split()
+    if len(command) != 3 or \
+            command[0].lower() != "mv" or \
+            command[1] == command[2]:
+        return
+
+    fileinfo1 = FileInfo(command[1])
```

Comment:
```suggestion
    source = FileInfo(command[1])
```
Not a really good name

Reply:
Corrected